src/Test/SparseVector/SparceVector.cs

```diff
@@ -0,0 +1,97 @@
+namespace SparseVector;
+
+public class Vector
+{
+    private Dictionary<int, float> vector;
+
+    public Vector()
+    {
+        this.vector = new Dictionary<int, float>();
+    } 
+
+    public void AddCoordinates(int index, float coordinates)
+    {
+        vector[index] = coordinates;
+    }   
+
+    public float GetCordinates(int index)
+    {
+        if (!vector.ContainsKey(index))
+        {
+            return 0;
+        }
+        else
+        {
+            return vector[index];
+        }
```

Comment:
```suggestion
return vector.ContainsKey(index) ? vector[index] : 0;
```
Это то же самое, но короче и понятнее